src/tests/api/backend/HappyPath.test.ts

```diff
@@ -143,51 +143,57 @@ describe("/documentSelection Endpoint", () => {
 	// for pdf_Preference and postal_address properties
 
-	// it.each([
-	// 	{ docSelectionData: dataUkDrivingLicencePrintedLetter },
-	// ])("Successful Request Tests - $PrintedLetter", async ({ docSelectionData }) => {
-	// 	const newf2fStubPayload = structuredClone(f2fStubPayload);
-	// 	const { sessionId } = await startStubServiceAndReturnSessionId(newf2fStubPayload);
-
-
-	// 	const postResponse = await postDocumentSelection(docSelectionData, sessionId);
-	// 	expect(postResponse.status).toBe(200);
-
-	// 	const personIdentityRecord = await getPersonIdentityRecordById(sessionId, constants.DEV_F2F_PERSON_IDENTITY_TABLE_NAME);
-	// 	try {
-	// 		expect(personIdentityRecord?.pdfPreference).toBe(dataUkDrivingLicence.pdf_preference);
-	// 	} catch (error) {
-	// 		console.error("Error validating PDF Preference from Person Identity Table", error);
-	// 		throw error;
-	// 	}
-	// });
-
-	// it.each([
-	// 	{ docSelectionData: dataUkDrivingLicencePreferredAddress },
-	// ])("Successful Request Tests - $PreferredAddress", async ({ docSelectionData }) => {
-	// 	const newf2fStubPayload = structuredClone(f2fStubPayload);
-	// 	const { sessionId } = await startStubServiceAndReturnSessionId(newf2fStubPayload);
-
-
-	// 	const docSelect = structuredClone(docSelectionData);
-	// 	docSelect.postal_address.preferredAddress = true;
-	// 	const postResponse = await postDocumentSelection(docSelectionData, sessionId);
-	// 	expect(postResponse.status).toBe(200);
-
-	// 	const personIdentityRecord = await getPersonIdentityRecordById(sessionId, constants.DEV_F2F_PERSON_IDENTITY_TABLE_NAME);
-
-	// 	try {
-	// 		expect(personIdentityRecord?.pdfPreference).toBe(docSelectionData.pdf_preference);
-	// 		const preferredAddress = personIdentityRecord?.addresses?.find(address => address.preferredAddress);
-	// 		expect(preferredAddress).toBeDefined();
-	// 		expect(preferredAddress?.postalCode).toBe(docSelectionData.postal_address.postalCode);
-	// 		expect(preferredAddress?.preferredAddress).toBe(true);
-	// 	} catch (error) {
-	// 		console.error("Error validating PDF and Address Preference from Person Identity Table", error);
-	// 		throw error;
-	// 	}
-
-	// });
+	it.each([
+		{ docSelectionData: dataUkDrivingLicencePrintedLetter },
+	])("Successful Request Tests - $PrintedLetter", async ({ docSelectionData }) => {
+		const newf2fStubPayload = structuredClone(f2fStubPayload);
+		const { sessionId } = await startStubServiceAndReturnSessionId(newf2fStubPayload);
+
+
+		const postResponse = await postDocumentSelection(docSelectionData, sessionId);
+		expect(postResponse.status).toBe(200);
+
+		const personIdentityRecord = await getPersonIdentityRecordById(sessionId, constants.DEV_F2F_PERSON_IDENTITY_TABLE_NAME);
+		try {
+			expect(personIdentityRecord?.pdfPreference).toBe(docSelectionData.pdf_preference);
+		} catch (error) {
+			console.error("Error validating PDF Preference from Person Identity Table", error);
+			throw error;
```

Comment:
Could this just be fail rather than exception so we clearly see why when the tests run